From ca7228c251f4ec32d2e5998376c411d54f5b05d4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cristian=20Rodr=C3=ADguez?= <crrodriguez@opensuse.org>
Date: Mon, 16 Jun 2014 01:13:44 -0400
Subject: [PATCH] systemd: Optionally, include efivarsfs module

It is, however not an error if this module can't be found.
---
 modules.d/98systemd/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/98systemd/module-setup.sh b/modules.d/98systemd/module-setup.sh
index cf1b2e71f1..dd82ecebe4 100755
--- a/modules.d/98systemd/module-setup.sh
+++ b/modules.d/98systemd/module-setup.sh
@@ -21,6 +21,7 @@ depends() {
 
 installkernel() {
     instmods autofs4 ipv6
+    instmods -s efivarfs
 }
 
 # called by dracut
-- 
1.9.3

